lib/trailhead_admin_profile.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
+        <members>Account</members>
+        <members>Campaign</members>
+        <members>Contact</members>
+        <members>Lead</members>
+        <members>Opportunity</members>
+        <members>npe01__OppPayment__c</members>
+        <members>npo02__Household__c</members>
+        <members>npo02__Opportunity_Rollup_Error__c</members>
+        <members>npe03__Custom_Field_Mapping__c</members>
+        <members>npe03__Recurring_Donation__c</members>
+        <members>npe4__Relationship__c</members>
+        <members>npe4__Relationship_Auto_Create__c</members>
+        <members>npe4__Relationship_Error__c</members>
+        <members>npe4__Relationship_Lookup__c</members>
+        <members>npe5__Affiliation__c</members>
+        <members>npsp__Account_Soft_Credit__c</members>
+        <members>npsp__Address_Verification_Settings__c</members>
+        <members>npsp__Address__c</members>
+        <members>npsp__Allocation__c</members>
+        <members>npsp__Allocations_Settings__c</members>
+        <members>npsp__Batch_Data_Entry_Settings__c</members>
+        <members>npsp__Batch__c</members>
+        <members>npsp__Customizable_Rollup_Settings__c</members>
+        <members>npsp__DataImportBatch__c</members>
+        <members>npsp__DataImport__c</members>
+        <members>npsp__Data_Import_Settings__c</members>
+        <members>npsp__Engagement_Plan_Task__c</members>
+        <members>npsp__Engagement_Plan_Template__c</members>
+        <members>npsp__Engagement_Plan__c</members>
+        <members>npsp__Error_Settings__c</members>
+        <members>npsp__Error__c</members>
+        <members>npsp__Fund__c</members>
+        <members>npsp__General_Accounting_Unit__c</members>
+        <members>npsp__Grant_Deadline__c</members>
+        <members>npsp__Household_Naming_Settings__c</members>
+        <members>npsp__Level__c</members>
+        <members>npsp__Levels_Settings__c</members>
+        <members>npsp__Opportunity_Naming_Settings__c</members>
+        <members>npsp__Partial_Soft_Credit__c</members>
+        <members>npsp__Relationship_Sync_Excluded_Fields__c</members>
+        <members>npsp__Schedulable__c</members>
+        <members>npsp__Trigger_Handler__c</members>
+        <name>CustomObject</name>
+    </types>
+    <types>
+        <members>Admin</members>
+        <name>Profile</name>
+    </types>
+    <version>43.0</version>
```

Comment:
```suggestion
    <version>47.0</version>
```

Reply:
Is this a `package.xml` versus a profile or are you just taking advantage of the fact that a package.xml can have a different name?

Reply:
It is ultimately a `package.xml` ; it is used by the `update_admin_profile` task.